packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -890,6 +1046,39 @@ describe('RichTextEditor', () => {
         }
     });
 
+    describe('`getMarkdown` with hard break backslashes should be same immediately after `setMarkdown`', () => {
+        const hardBreakMarkdownStrings: { name: string }[] = [
+            { name: '**bold**\\\n*Italics*' },
+            { name: '* list\\\n  hard break content\n\n* list' },
+            {
+                name: '* list\\\n  hard break content\n\n* list\n\n  * nested list\\\n    nested hard break content'
+            },
+            { name: '1. list\\\n   hard break content\n\n2. list' },
+            {
+                name: '1. list\\\n   hard break content\n\n2. list\n\n   1. nested list\\\n      nested hard break content'
+            }
+        ];
```

Comment:
Consider having a value here to keep the test case title simple as mentioned in [other comment](https://github.com/ni/nimble/pull/1517#discussion_r1323261302)